src/main/play-doc/developer/ContinuousDeliverySetup.md

```diff
@@ -0,0 +1,224 @@
+# Continuous Delivery Setup
+
+Continuous Delivery is a ConductR feature which allows bundles to deployed to a ConductR cluster upon being published to Bintray.
```

Comment:
We should introduce what CD is and why the reader should care. Refer to Wikipedia perhaps, or extract from our original design doc. Also describe how CD works in the context of ConductR i.e. the rolling upgrade procedure. I find it quite exciting. Make mention of how we preserve bundle configurations containing sensitive data.